site/_data/docs/extensions/toc.yml

```diff
@@ -51,6 +51,7 @@
     - url: /docs/extensions/mv3/service_workers/basics
     - url: /docs/extensions/mv3/service_workers/service-worker-lifecycle
     - url: /docs/extensions/mv3/service_workers/events
+    - url: /docs/extensions/mv3/tut_websockets
```

Comment:
would it make sense for there to be a `/docs/extensions/mv3/tut` or `/docs/extensions/mv3/tutorials` section, rather than the `tut` prefix?

Reply:
Definitely, we should fix this once we move to a new IA

Reply:
👍 - are we tracking todos for that anywhere?

Reply:
no, I don't think it's needed for this case though.